From 17d4ce4aa5067849ca76e4b1e689b21771322ca0 Mon Sep 17 00:00:00 2001
From: Down to the Wire <8404598+wireddown@users.noreply.github.com>
Date: Sun, 4 Jun 2023 23:16:55 -0700
Subject: @wireddown: Add thumbprint and signpost

---
 Marlin/Configuration.h | 6 +++---
 Marlin/Version.h       | 8 ++++----
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/Marlin/Configuration.h b/Marlin/Configuration.h
index 9fb2ed7..23e3fb7 100644
--- a/Marlin/Configuration.h
+++ b/Marlin/Configuration.h
@@ -60,11 +60,11 @@
  */
 
 // @section info
 
 // Author info of this build printed to the host during boot and M115
-#define STRING_CONFIG_H_AUTHOR "3DPrintMill" // Who made the changes.
+#define STRING_CONFIG_H_AUTHOR "@wireddown" // Who made the changes.
 #define CUSTOM_VERSION_FILE Version.h // Path from the root directory (no quotes)
 
 /**
  * *** VENDORS PLEASE READ ***
  *
@@ -135,15 +135,15 @@
 
 // Enable the Bluetooth serial interface on AT90USB devices
 //#define BLUETOOTH
 
 // Name displayed in the LCD "Ready" message and Info menu
-#define CUSTOM_MACHINE_NAME "3DPrintMill"
+#define CUSTOM_MACHINE_NAME "Camina"
 
 // Printer's unique ID, used by some programs to differentiate between machines.
 // Choose your own or use a service like https://www.uuidgenerator.net/version4
-//#define MACHINE_UUID "00000000-0000-0000-0000-000000000000"
+#define MACHINE_UUID "CA319A00-0000-0000-0000-00BE174103D4"
 
 // @section stepper drivers
 
 /**
  * Stepper Drivers
diff --git a/Marlin/Version.h b/Marlin/Version.h
index 8287902..83cb3e1 100644
--- a/Marlin/Version.h
+++ b/Marlin/Version.h
@@ -26,48 +26,48 @@
 ////////////////////////////
 
 /**
  * Marlin release version identifier
  */
-//#define SHORT_BUILD_VERSION "2.1.2.1"
+#define SHORT_BUILD_VERSION "2.1.2.1+wireddown"
 
 /**
  * Verbose version identifier which should contain a reference to the location
  * from where the binary was downloaded or the source code was compiled.
  */
-//#define DETAILED_BUILD_VERSION SHORT_BUILD_VERSION
+#define DETAILED_BUILD_VERSION "09d0b4d15 + pull/1"
 
 /**
  * The STRING_DISTRIBUTION_DATE represents when the binary file was built,
  * here we define this default string as the date where the latest release
  * version was tagged.
  */
 //#define STRING_DISTRIBUTION_DATE "2023-05-16"
 
 /**
  * Defines a generic printer name to be output to the LCD after booting Marlin.
  */
 //#define MACHINE_NAME "3D Printer"
 
 /**
  * The SOURCE_CODE_URL is the location where users will find the Marlin Source
  * Code which is installed on the device. In most cases —unless the manufacturer
  * has a distinct Github fork— the Source Code URL should just be the main
  * Marlin repository.
  */
-//#define SOURCE_CODE_URL "github.com/MarlinFirmware/Marlin"
+#define SOURCE_CODE_URL "github.com/wireddown/GCode-Machine-Configurations"
 
 /**
  * Default generic printer UUID.
  */
 //#define DEFAULT_MACHINE_UUID "cede2a2f-41a2-4748-9b12-c55c62f367ff"
 
 /**
  * The WEBSITE_URL is the location where users can get more information such as
  * documentation about a specific Marlin release.
  */
-//#define WEBSITE_URL "marlinfw.org"
+#define WEBSITE_URL "github.com/wireddown"
 
 /**
  * Set the vendor info the serial USB interface, if changable
  * Currently only supported by DUE platform
  */
-- 
2.41.0.windows.1

