From 8937407477edec062cd03b1954c179bee48c3109 Mon Sep 17 00:00:00 2001
From: Down to the Wire <8404598+wireddown@users.noreply.github.com>
Date: Sun, 30 Apr 2023 16:03:41 -0700
Subject: [PATCH 12/16] Enable hotend parking for SD cancel

---
 Marlin/Configuration.h     | 2 +-
 Marlin/Configuration_adv.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Marlin/Configuration.h b/Marlin/Configuration.h
index a461b12e91..260bc65430 100644
--- a/Marlin/Configuration.h
+++ b/Marlin/Configuration.h
@@ -2087,7 +2087,7 @@
  *    P1  Raise the nozzle always to Z-park height.
  *    P2  Raise the nozzle by Z-park amount, limited to Z_MAX_POS.
  */
-//#define NOZZLE_PARK_FEATURE
+#define NOZZLE_PARK_FEATURE
 
 #if ENABLED(NOZZLE_PARK_FEATURE)
   // Specify a park position as { X, Y, Z_raise }
diff --git a/Marlin/Configuration_adv.h b/Marlin/Configuration_adv.h
index 5a0a544275..bb65bb1f20 100644
--- a/Marlin/Configuration_adv.h
+++ b/Marlin/Configuration_adv.h
@@ -1453,7 +1453,7 @@
 
   //#define MEDIA_MENU_AT_TOP               // Force the media menu to be listed on the top of the main menu
 
-  #define EVENT_GCODE_SD_ABORT "G28XY"      // G-code to run on SD Abort Print (e.g., "G28XY" or "G27")
+  #define EVENT_GCODE_SD_ABORT "G27"        // G-code to run on SD Abort Print (e.g., "G28XY" or "G27")
 
   #if ENABLED(PRINTER_EVENT_LEDS)
     #define PE_LEDS_COMPLETED_TIME  (30*60) // (seconds) Time to keep the LED "done" color before restoring normal illumination
-- 
2.40.0.windows.1

