From 5f6835676959468dff30cf5e5be5c2cb3dc83680 Mon Sep 17 00:00:00 2001
From: Down to the Wire <8404598+wireddown@users.noreply.github.com>
Date: Sun, 4 Jun 2023 22:45:39 -0700
Subject: CR30-Users: User-specific preheat settings

---
 Marlin/Configuration.h | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/Marlin/Configuration.h b/Marlin/Configuration.h
index 6c147c6..0910bcf 100644
--- a/Marlin/Configuration.h
+++ b/Marlin/Configuration.h
@@ -2214,21 +2214,21 @@
 // @section temperature
 
 //
 // Preheat Constants - Up to 10 are supported without changes
 //
-#define PREHEAT_1_LABEL       "PLA"
-#define PREHEAT_1_TEMP_HOTEND 185
-#define PREHEAT_1_TEMP_BED     55
-#define PREHEAT_1_TEMP_CHAMBER 30
-#define PREHEAT_1_FAN_SPEED     0 // Value from 0 to 255
-
-#define PREHEAT_2_LABEL       "ABS"
-#define PREHEAT_2_TEMP_HOTEND 240
-#define PREHEAT_2_TEMP_BED     70
-#define PREHEAT_2_TEMP_CHAMBER 35
-#define PREHEAT_2_FAN_SPEED     0 // Value from 0 to 255
+#define PREHEAT_1_LABEL       "PLA"  // Set as you like
+#define PREHEAT_1_TEMP_HOTEND 185    // Set as you like
+#define PREHEAT_1_TEMP_BED     55    // Set as you like
+#define PREHEAT_1_TEMP_CHAMBER 30    // Set as you like
+#define PREHEAT_1_FAN_SPEED     0    // Value from 0 to 255
+
+#define PREHEAT_2_LABEL       "ABS"  // Set as you like
+#define PREHEAT_2_TEMP_HOTEND 240    // Set as you like
+#define PREHEAT_2_TEMP_BED     70    // Set as you like
+#define PREHEAT_2_TEMP_CHAMBER 35    // Set as you like
+#define PREHEAT_2_FAN_SPEED     0    // Value from 0 to 255
 
 // @section motion
 
 /**
  * Nozzle Park
-- 
2.41.0.windows.1

